tachyon/crypto/commitments/kzg/kzg.h

```diff
@@ -148,12 +148,14 @@ class KZG {
         // NOLINTNEXTLINE(readability/braces)
       } else if constexpr (std::is_same_v<Commitment,
                                           math::AffinePoint<Curve>>) {
-        batch_commitments.resize(cpu_batch_commitments_.size());
+        batch_commitments.resize(cpu_batch_commitments_.size(),
+                                 Commitment::Zero());
         CHECK(
             Bucket::BatchNormalize(cpu_batch_commitments_, &batch_commitments));
         cpu_batch_commitments_.clear();
       } else {
-        batch_commitments.resize(cpu_batch_commitments_.size());
+        batch_commitments.resize(cpu_batch_commitments_.size(),
```

Comment:
It seems like we don't need to initialize the values, since `BatchNormalize` will be filled with the values. But just add a comment that why this is safe.